src/store/metadata/stub/content/editions/versions/age-16-to-64-years-level_2023-05.json

```diff
@@ -31,18 +31,6 @@
                   "titles": "ONS Geography Code",
                   "description": "The statistical geography of the observation being measured."
                 },
-                {
-                  "name": "areanm",
-                  "datatype": "string",
-                  "titles": "Area Name",
```

Comment:
Please revert change and include Area Name
change title to "Statistical Geography"